error TS4082: Default export of the module has or is using private name 'InternalTime'

**Describe the bug**
Latest version of TypeScript (5.7.2) and vue-tsc (2.2.0) gives the following error:
error TS4082: Default export of the module has or is using private name 'InternalTime'
This is because InternalTime is not exported in index.d.ts, although it is used in other exported structs

**To Reproduce**
Try to build with TypeScript (5.7.2) and vue-tsc (2.2.0)

**Expected behavior**
InternalTime should be exported in index.d.ts

```
error TS4082: Default export of the module has or is using private name 'InternalTime'.

852 export default (await import('vue')).defineComponent({
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
853 setup() {
    ~~~~~~~~~
... 
872 __typeEl: {} as __VLS_TemplateResult['rootEl'],
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
873 });
    ~~~
```

Exporting the type fixes the issue.
```diff
-interface InternalTime {
+export interface InternalTime {
    hours: number | number[];
    minutes: number | number[];
    seconds: number | number[];
}
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -329,7 +329,7 @@ export interface PublicMethods extends MethodOptions {
 }
 
 type InternalModelValue = Date | Date[] | null;
-interface InternalTime {
+export interface InternalTime {
     hours: number | number[];
     minutes: number | number[];
     seconds: number | number[];

